python/distant_vfx/filesystem.py

```diff
@@ -33,6 +34,11 @@ def _chunk_items(items, chunk_size):
             result.append(list(item.keys()))
         return result
 
+    def _scandir(self, path):
```

Comment:
So `os.scandir` doesn't exist in python2.

`os.scandir` appears to be a simple `os.walk` but returning pathlib.Path objects.  
I think all of my replacements have updated it to return just filepaths.

**Note** It is not returning any directories, is that correct?

Reply:
I may be wrong but I think `os.walk` actually uses `os.scandir` in 3.5+, not the other way around. I think in your `_scandir` implementation `os.walk` is recursively searching through subfolders, which we actually don't want here. `os.scandir` scans only the root folder, not subfolders, it's more like `os.listdir`. Also, we do need it to return directories, most of the time this `Chunker` is being used to sort folders of set data. 

Maybe something like this can work?
`
from pathlib import Path

items = os.lisdir(path)
for i in items:
    item = os.path.join(path, i)
    ...
`

Let me know your thoughts.

Reply:
Yeah that looks good to me! Thanks for the clarity on what scandir does.

I'll update the PR :)